index.html

```diff
@@ -5,8 +5,310 @@
+              <div class="slide-block">
+                <h2 class="slider-header">Нежный пломбир с клубничным джемом</h2>
+                <p class="slider-text">Натуральное мороженое из свежих сливок и молока с вкуснейшим клубничным джемом – это идеальный десерт для всей семьи.</p>
+                <a href="catalog.html" class="button">Заказать</a>
+              </div>
+              <img src="https://via.placeholder.com/306x507" alt="Мороженое-рожок с клубничным джемом" class="slide-image" width="306" height="507">
+            </li>
+            <li class="slider-item" id="slide2">
+              <div class="slide-block">
+                <h2 class="slider-header">Сливочное мороженое со вкусом банана</h2>
+                <p class="slider-text">Сливочное мороженое с ярким банановым вкусом подарит вам свежесть и наслаждение даже в самый жаркий летний день.</p>
+                <a href="catalog.html" class="button">Заказать</a>
+              </div>
+              <img src="https://via.placeholder.com/66x150" alt="Мороженое-рожок с бананом" class="slide-image" width="66" height="150">
+            </li>
+            <li class="slider-item" id="slide3">
+              <div class="slide-block">
+                <h2 class="slider-header">Карамельный пломбир с маршмеллоу</h2>
+                <p class="slider-text">Необычный сладкий дессерт с карамельный топпингом и кусочками зефира завоюет сердца сладкоежек всех возрастов.</p>
+                <a href="catalog.html" class="button">Заказать</a>
+              </div>
+              <img src="https://via.placeholder.com/66x150" alt="Мороженое-рожок с карамельный топпингом и кусочками зефира" class="slide-image" width="66" height="150">
+            </li>
+          </ul>
+          <div class="slider-controls">
+            <label for="product-1">1</label>
+            <label for="product-2">2</label>
+            <label for="product-3">3</label>
+          </div>
+          <div class="slider-nav">
+            <button class="slider-button previous" type="button">
+              <span class="visually-hidden">Предыдущий товар</span>
+            </button>
+            <button class="slider-button next" type="button">
+              <span class="visually-hidden">Следующий товар</span>
+            </button>
+          </div>
+          <ul class="social-list">
+            <li class="social-item">
+                <a href="#" class="social-link"><img src="https://via.placeholder.com/24x24" alt="Иконка vkontakte" width="24" height="24"></a>
+            </li>
+            <li class="social-item">
+                <a href="#" class="social-link"><img src="https://via.placeholder.com/24x24" alt="Иконка facebook" width="24" height="24"></a>
+            </li>
+            <li class="social-item">
+                <a href="#" class="social-link"><img src="https://via.placeholder.com/24x24" alt="Иконка instagram" width="24" height="24"></a>
+            </li>
+          </ul>
```

Comment:
Я рекомендую всегда при наличии тега внутри другого тега делать отдельную строку для внутреннего7
Так код читаемее:
```suggestion
          <ul class="social-list">
            <li class="social-item">
              <a href="#" class="social-link">
                <img src="https://via.placeholder.com/24x24" alt="Иконка vkontakte" width="24" height="24">
              </a>
            </li>
            <li class="social-item">
              <a href="#" class="social-link">
                <img src="https://via.placeholder.com/24x24" alt="Иконка facebook" width="24" height="24">
              </a>
            </li>
            <li class="social-item">
              <a href="#" class="social-link"><img src="https://via.placeholder.com/24x24" alt="Иконка instagram" width="24" height="24">
              </a>
            </li>
          </ul>
```

Reply:
исправлено